FastSimulation/ParticlePropagator/plugins/MagneticFieldMapESProducer.h

```diff
@@ -14,7 +17,9 @@ class MagneticFieldMapESProducer : public edm::ESProducer {
   std::unique_ptr<MagneticFieldMap> produce(const MagneticFieldMapRecord &);
 
 private:
-  std::string _label;
+  const std::string _label;
```

Comment:
Since you are touching it anyhow, you can respect the convention of the other class members and move the underscore at the end